src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  &__box {
+    display: flex;
+    width: 944px;
```

Comment:
Width for the catalog should be calculated depending on the card number, card width and indents between them